test/org/freedesktop/gstreamer/PadTest.java

```diff
@@ -263,6 +262,9 @@ public PadProbeReturn dataReceived(Pad pad, Buffer buffer) {
         // push data
         res = src.push(buf2);
         assertNotSame("data_prober.probeData() should not have been called", buf2, b.get());
+        
+        elem.stop();
```

Comment:
OK. Causing a problem?

Reply:
Trace from test run in master branch :

**Trying to dispose element src, but it is in PLAYING instead of the NULL state.
You need to explicitly set elements to the NULL state before
dropping the final reference, to allow them to clean up.
This problem may also be caused by a refcounting bug in the
application or some element.**

Reply:
Ah, OK, can see why that's an issue.  Thanks.